src/change_stream.ts

```diff
@@ -235,19 +234,21 @@ export class ChangeStream extends EventEmitter {
     // Create contained Change Stream cursor
     this.cursor = createChangeStreamCursor(this, options);
 
+    const cursorStream = this.cursor._stream;
+    if (!cursorStream) throw new MongoError('No stream on cursor');
```

Comment:
A `CursorStream` should only ever be created if a user is attempting to use the change stream as a stream. I'm actually wondering if we should continue the breaking changes here:
- let's make `Cursor` an `EventEmitter` as we discussed in team meeting yesterday
- let's remove any logic from `ChangeStream` about it being an explicit `Readable`, and use the same behavior as cursor here: you must call `stream()` on the change stream in order to use it as a `Readable` stream. I think this will cut out a lot of complexity of the implementation since you won't have to manage an internal stream anymore. What do you think?

Reply:
Agreed, this makes sense and will definitely simplify the implementation. I think the consistency is also good; it should be easy for users to get used to calling `stream()` when they need a stream.